maturity-levels/maturity-level-decriptors.md

```diff
@@ -0,0 +1,117 @@
+[//]: # (COPYRIGHT)
+[//]: # (RiskFrame.ai - AI Risk Management and Resilience Framework)
+[//]: # (Copyright (C) 2024 RiskFrame.ai https://www.riskframe.ai https://github.com/riskframe/ai-rmm)
+[//]: # (SOFTWARE LICENSE)
+[//]: # (This file is part of AI-RMM, which is distributed under GNU General Public License V3. See LICENSE.txt to get a full copy.)
+
+# AI-RMM Maturity Level Descriptors
+
+## Understanding AI-Aware Organizational Resilience 
```

Comment:
This entire block is a duplicate of the Readme and feels completely redundant. We can start this doc from the "Progressive Maturity Framework"